### PR TITLE
Tests: Adding some behave tests for timestamp

### PR DESCRIPTION
**Pull request by ahmadnazeri**
# Description
Adding some tests for timestamp columns within Features and Labels as well as removing the `include_label_timestamp` from `client.training_set`

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have fixed any merge conflicts

### CODE REVIEW

File: provider/k8s.go [hunk 1]
```diff
@@ -1164,6 +1165,15 @@ func (iter *FileStoreFeatureIterator) Next() bool {
 
 func castToTimestamp(timestamp interface{}) (time.Time, error) {
 	if ts, ok := timestamp.(time.Time); !ok {
+		// TODO: this is a temporary fix for the timestamp issue
```
Comment (simba-git) [inline provider/k8s.go:1168]: Your call, but I'd do a:
switch timestamp.(type) {
case time.Time:
case string:
default:
}

**Review by simba-git — APPROVED** [review]

File: provider/k8s.go (hunk 1 above)
Comment (aolfat) [reply provider/k8s.go:1168]: i agree w/ simba's, easier to read

File: tests/end_to_end/features/steps/model_registration.py [hunk 2]
```diff
@@ -28,7 +28,26 @@ def step_impl(context, offline_provider_type):
...
-        raise NotImplementedError("Spark registration is not implemented")
+        """
+        Need to register spark with s3 and databricks
+        TODO: modify this so you can create spark with different Filestore & Executor
+        """
+
+        # create a s3 filestore
+        context.execute_steps(
+            """
+            when I register "s3" filestore with bucket "featureform-spark-testing" and root path "behave"
```
Comment (aolfat) [inline tests/end_to_end/features/steps/model_registration.py:39]: interesting

File: tests/end_to_end/features/features_with_timestamp.feature [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Feature: Features with Timestamp
```
Comment (aolfat) [inline tests/end_to_end/features/features_with_timestamp.feature:1]: use the @use_autovariants tag here

**Pull request merged**